docs/functions/theme.rst

```diff
@@ -111,6 +111,13 @@ specific text and formatting may change.
       Return code matches data function.
       Return method changed from stdout.
 
+.. function:: _lp_container() -> var:lp_container
```

Comment:
Incorrect function name and return variable.

Reply:
fixed